Consolidate to one quarterly event strategy meeting (#42408)
fleetdm/fleet: handbook/marketing/event-execution.md

  - Who from Fleet will attend?
  - Which talk proposal will Fleet submit?
  - Estimated budget, including sponsorship or airfare, and lodging for attendees.

> 2. Set up and attend a 30m quarterly event strategy meeting with the CMO, Head of Demand Generation, or Manager of Training and Enablement.

Reviewer: ```suggestion
2. Attend the 30m quarterly event strategy meeting with the CMO, Head of Demand Generation, and Marketing Campaign Manager.
```
